CHANGELOG.md

```diff
@@ -7,7 +7,12 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ## [Unreleased]
 
-- Added support for Clarity 3
+## [3.0.0.0.0]
+
+### Added
+
+- **Nakamoto consensus rules, activating in epoch 3.0 at block 867,867**
```

Comment:
Perhaps a reference to the SIP would be in order?

Reply:
@obycode https://github.com/stacksgov/sips/blob/main/sips/sip-021/sip-021-nakamoto.md